source/uwp/Renderer/lib/XamlBuilder.h

```diff
@@ -150,6 +151,14 @@ AdaptiveNamespaceStart
         void PopulateImageFromUrlAsync(_In_ ABI::Windows::Foundation::IUriRuntimeClass* imageUrl, T* imageControl);
         void FireAllImagesLoaded();
         void FireImagesLoadingHadError();
+        // Keeping it alive cause I dont know what this is
+        /*
+        void BuildPanelChildren(
+            _In_ ABI::Windows::Foundation::Collections::IVector<ABI::AdaptiveCards::XamlCardRenderer::IAdaptiveCardElement*>* children,
+            _In_ ABI::Windows::UI::Xaml::Controls::IGrid* parentGrid,
+            std::shared_ptr<std::vector<InputItem>> inputElements,
+            _In_ std::function<void(ABI::Windows::UI::Xaml::IUIElement* child)> childCreatedCallback);
+        */
```

Comment:
This looks like an older version of the BuildPanelChildren function (now defined below as a static). Please remove.